pkg/kubevirt/core/core.go

```diff
@@ -112,6 +112,13 @@ func (p PluginSPIImpl) CreateMachine(ctx context.Context, machineName string, pr
 		}
 	}
 
+	var interfaces []kubevirtv1.Interface
```

Comment:
Nit: can be grouped:
```go
var (
  interfaces     []kubevirtv1.Interface
  networks       []kubevirtv1.Network
  networkData string
)

Reply:
I am a strong proponent of "declaration at the point of initialization" and in Go, variable declaration + initialization with `:=` as opposed to `var` blocks. IMO this makes the code much more expressive and easier to read. In the Gardener codebase, both styles are used to a certain extent, and so it doesn't provide a strong hint as to which one should be preferred.
To avoid a religious discussion, I refactored the code to avoid having to initialize these variables with empty values here, which obsoletes the need for a block.